Modules/Common/EVMMainModule.php

```diff
@@ -174,6 +174,13 @@ final public function pre_process_block($block_id)
                 }
             }
 
+            if (in_array(EVMSpecialFeatures::ProcessValidatorSeparately, $this->extra_features))
```

Comment:
Should we really implement this logic in the main module? As it is bor specific case better to move it somehow to PolygonMainModule or at least the request params should be in PolygonMainModule.
Imagin if the X EVM blockchain would have the same logic, but the method name won't be `bor_getAuthor`

Reply:
Yes, that's the only good way under the current architecture. Otherwise we'd need to copy the entire `pre_process_block` for this small differece.

Reply:
Clearer now, awesome!:+1: :heart_eyes: